knimin/config.txt.example

```diff
@@ -3,9 +3,9 @@ debug = True
 help_email =
 
 [postgres]
-USER =
+USER = postgres
 PASSWORD =
-DATABASE =
+DATABASE = ag_test
```

Comment:
Well, itd be annoying at least... Okay, not blocking
On Oct 12, 2015 11:54 AM, "Joshua Shorenstein" <notifications@github.com>
wrote:

> In knimin/config.txt.example
> <https://github.com/biocore/labadmin/pull/6#discussion_r41788847>:
>
> >  PASSWORD =
> > -DATABASE =
> > +DATABASE = ag_test
>
> The database made by running ag_make_test i called ag_test, so can't
> really rename.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/biocore/labadmin/pull/6/files#r41788847>.
>